db/schema.rb

```diff
@@ -219,8 +219,8 @@
     t.integer "inbox_id", null: false
     t.integer "status", default: 0, null: false
     t.integer "assignee_id"
-    t.datetime "created_at", null: false
```

Comment:
@sojan-official Why was this change done?

Reply:
I think this got crept in unintentionally. let me raise a PR fixing this.